py3-supported-python.yaml

```diff
@@ -28,6 +35,38 @@ subpackages:
         - python-3.12-base-dev
         - python-3.13-base-dev
 
+  - range: py-versions
+    name: py${{range.key}}-build-base
+    description: ${{vars.pypi-package}} core/base build packages.
+    dependencies:
+      runtime:
+        - py${{range.key}}-build
+        - py${{range.key}}-installer
+        - py${{range.key}}-pip-base
+        - py${{range.key}}-setuptools
```

Comment:
You probably want setuptools-scm by default here too to ensure version correctness from git-checkout.

Reply:
I did consider setuptools-scm as you're right it is useful.
i left it out because of its dependency tree. just checked, and it just adds typping-extensions, so i'll add it.